Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -366,20 +366,32 @@ def OnMouseWheelBackward(self, interactor, event):
         if self.GetViewerEvent("SHOW_LINE_PROFILE_EVENT"):
             self.DisplayLineProfile(interactor, event, True)
 
-    def AutoWindowLevel(self):
-        # reset color/window
+    def AutoWindowLevelOnSliceRange(self, update_slice=True):
+        '''Auto-adjusts window-level for the slice, based on the 5 and 95th percentiles of the current slice.'''
```

Comment:
How do you know it's 5th 95th percentiles?
Wouldn't it be better to set the percentiles in this method?